src/Module/Api/Mastodon/Accounts/Block.php

```diff
@@ -41,26 +41,17 @@ protected function post(array $request = [])
 			DI::mstdnError()->UnprocessableEntity();
 		}
 
-		$owner = User::getOwnerDataById($uid);
```

Comment:
Why remove the local user existence check?

Reply:
We don't need it. We fetch the `$uid` some lines before. And this can only work when we have got this user.